eng/versioning/version_client.txt

```diff
@@ -100,6 +100,13 @@ com.microsoft.azure:spring-cloud-azure-feature-management;1.2.8-beta.1;1.2.8-bet
 com.microsoft.azure:spring-cloud-azure-keyvault-config;1.2.8-beta.1;1.2.8-beta.1
 com.microsoft.azure:spring-cloud-azure-messaging;1.2.8-beta.1;1.2.8-beta.1
 com.microsoft.azure:spring-cloud-azure-storage;1.2.8-beta.1;1.2.8-beta.1
+com.microsoft.azure:spring-cloud-starter-azure-appconfiguration-config;1.2.8-beta.1;1.2.8-beta.1
+com.microsoft.azure:spring-cloud-starter-azure-eventhubs-kafka;1.2.8-beta.1;1.2.8-beta.1
+com.microsoft.azure:spring-cloud-starter-azure-eventhubs;1.2.8-beta.1;1.2.8-beta.1
+com.microsoft.azure:spring-cloud-starter-azure-servicebus;1.2.8-beta.1;1.2.8-beta.1
+com.microsoft.azure:spring-cloud-starter-azure-storage-queue;1.2.8-beta.1;1.2.8-beta.1
+com.microsoft.azure:spring-starter-azure-cache;1.2.8-beta.1;1.2.8-beta.1
+com.microsoft.azure:spring-starter-azure-storage;1.2.8-beta.1;1.2.8-beta.1
```

Comment:
@saragluna these artifacts need to get added to the work item for renaming.

Reply:
I added in this issue: https://github.com/Azure/azure-sdk-for-java/issues/13523